jujugui/static/gui/src/app/assets/css/_variables.scss

```diff
@@ -20,6 +19,16 @@ $color-information:      #335280 !default;
 
 $color-input-shadow:     rgba($color-x-dark, .12) !default;
 
+// Global spacing settings
+$sp-xx-small: 1rem * .25 !default;
```

Comment:
What are these? Where are they used?

Reply:
Removed, they were used in one of the new vanilla patterns that I copied over but I've just replaced them in place.